tests/keras_tests/function_tests/test_hmse_error_method.py

```diff
@@ -110,52 +109,58 @@ def _run_node_verification(node_type):
                 node.candidates_quantization_cfg[0].weights_quantization_cfg.get_attr_config(KERNEL))
             self.assertTrue(kernel_attr_qparams.weights_quantization_params.get(param_name) is not None,
                             f"Expecting {node_type} node parameters {param_name} to be initialized.")
-
             expected_hessian_request = HessianScoresRequest(mode=HessianMode.WEIGHTS,
                                                             granularity=HessianScoresGranularity.PER_ELEMENT,
+                                                            data_loader=None,
+                                                            n_samples=1,
                                                             target_nodes=[node])
-
-            self.assertTrue(self.his.count_saved_scores_of_request(expected_hessian_request)[node] > 0,
-                            f"No Hessian-based scores were computed for node {node}, "
-                            "but expected parameters selection to run with HMSE.")
+            # check hessians have been precomputed (dataloader=None fetches from cache)
+            hess = self.his.fetch_hessian(expected_hessian_request)
+            self.assertTrue(hess[node.name].shape[0] == 1)
```

Comment:
add a message to this assertion, so it would be simpler to understand what was the expected behavior (None dataloader should fetch from cache) in case it fails